client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -88,34 +88,34 @@
 public class RssShuffleManager extends RssShuffleManagerBase {
 
   private static final Logger LOG = LoggerFactory.getLogger(RssShuffleManager.class);
-  private final String clientType;
-  private final long heartbeatInterval;
-  private final long heartbeatTimeout;
-  private AtomicReference<String> id = new AtomicReference<>();
-  private SparkConf sparkConf;
-  private final int dataReplica;
-  private final int dataReplicaWrite;
-  private final int dataReplicaRead;
-  private final boolean dataReplicaSkipEnabled;
-  private final int dataTransferPoolSize;
-  private final int dataCommitPoolSize;
-  private ShuffleWriteClient shuffleWriteClient;
-  private final Map<String, Set<Long>> taskToSuccessBlockIds;
-  private final Map<String, Set<Long>> taskToFailedBlockIds;
-  private ScheduledExecutorService heartBeatScheduledExecutorService;
-  private boolean heartbeatStarted = false;
-  private boolean dynamicConfEnabled = false;
-  private final ShuffleDataDistributionType dataDistributionType;
-  private final int maxConcurrencyPerPartitionToWrite;
-  private String user;
-  private String uuid;
-  private Set<String> failedTaskIds = Sets.newConcurrentHashSet();
-  private DataPusher dataPusher;
-
-  private final Map<Integer, Integer> shuffleIdToPartitionNum = Maps.newConcurrentMap();
-  private final Map<Integer, Integer> shuffleIdToNumMapTasks = Maps.newConcurrentMap();
-  private ShuffleManagerGrpcService service;
-  private GrpcServer shuffleManagerServer;
+  protected final String clientType;
```

Comment:
why change all field to `protected`? If this is necessary, it is also need to add some code comments to explain

Reply:
Follow the suggestion